blocks/content-list/content-list.js

```diff
@@ -0,0 +1,63 @@
+import { div } from '../../scripts/dom-builder.js';
+import { toCamelCase } from '../../scripts/aem.js';
+import listArticles from '../article-list/article-list.js';
+
+function extractFilterAttribues(filterInfo) {
+  const filterId = filterInfo?.firstElementChild?.textContent?.toLowerCase();
+  let filterValue = filterInfo?.lastElementChild?.textContent?.toLowerCase();
+  if (filterId && filterValue) {
+    filterValue = filterValue.replace('*', '');
+    return { name: toCamelCase(filterId), value: filterValue.split(',').map((value) => value.trim()) };
+  }
+  return null;
+}
+
+function filter(entry, attributes) {
```

Comment:
Minor: its bit confusing if we are passing whole entry object or just an entry field as argument `entry`

Reply:
Named it differently :)